src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1190,6 +1190,14 @@ qlonglong TorrentImpl::timeSinceActivity() const
         : std::min(upTime, downTime);
 }
 
+qlonglong TorrentImpl::timeLastActivity() const
```

Comment:
Seems you are doing workaround for webapi, so please wrap/move it to a local lambda, look at `adjustQueuePosition()` in serialize_torrent.cpp for an example.

Reply:
done.